### PR TITLE
Fix: set links to inherit from footer color mixin (fixes #18)

### LINKED ISSUE #18: Links don't inherit from the footer color mixin
### Subject of the issue
Links don't inherit from the [footer color mixin](https://github.com/cgkineo/adapt-kineo-footer/blob/2e97933de64cfb2976cb8acdfd923a33bfaa4861/less/footer.less#L62). See **Test link** in the screen shot below.

link_color_not_inherited

### Expected behaviour
Link should inherit `color: #fff` (`color: @@color-inverted` in the mixin).

### Actual behaviour
Links inherit from the browser instead.

### PR DESCRIPTION
**Pull request by kirsty-hames**
Fixes https://github.com/cgkineo/adapt-kineo-footer/issues/18

### CODE REVIEW

**Review by guywillis — APPROVED** [review]
👀

File: less/footer.less [hunk 1]
```diff
@@ -62,6 +62,10 @@
 .footer-color.@{color} {
   background-color: @@color;
   color: @@color-inverted;
+
+  .footer__body a {
+    color: @@color-inverted;
```
Comment (swashbuck) [inline less/footer.less:67]: @kirsty-hames Could also use 'inherit' here.

**Review by swashbuck — APPROVED** [review]
👍

File: less/footer.less (hunk 1 above)
Comment (kirsty-hames) [reply less/footer.less]: Good shout @swashbuck - amended in [626f9b4](https://github.com/cgkineo/adapt-kineo-footer/pull/19/commits/626f9b485575e5737d8303cf84b63d3be24bb65b)

**Pull request merged**